src/test/java/io/github/thebusybiscuit/slimefun4/testing/mocks/DamageableMock.java

```diff
@@ -0,0 +1,32 @@
+package io.github.thebusybiscuit.slimefun4.testing.mocks;
+
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.DamageableItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class DamageableMock extends SlimefunItem implements DamageableItem {
+    boolean itemDamageable;
```

Comment:
Formatting / missing access modifiers

```suggestion

    private final boolean isDamageable;
```